crates/transpiler/src/target/mod.rs

```diff
@@ -1551,7 +1546,7 @@ impl Target {
                 }
                 params.len()
             }
-            None => {
+            _ => {
                 return Err(TargetError::InvalidKey(format!(
                     "{name} is not an instruction in the target."
                 )));
```

Comment:
This error message only makes sense in the `None` case, but the wildcard is masking it.  The `Some(TargetOperation::Variadic)` case should be going to a different error message, at a minimum.

Reply:
Fixed in 5ffb2115b8.